scripts/serie_update.py

```diff
@@ -668,6 +617,17 @@ def apply_zephyr_patch(self):
             )
         )
 
+        # apply dos2unix to whole zephyr hal serie sub directory
+        for child in self.zephyr_module_serie_path.glob("**/*"):
+            if child.is_file:
+                self.os_cmd(("dos2unix", child), cwd=self.zephyr_module_serie_path)
```

Comment:
Because I am working on windows (with git-bash), and at some point is show delta linked to CR/LF (I don't remember exactly but I think it depends on stm32 series repo).